pkg/config/config_test.go

```diff
@@ -226,6 +234,14 @@ func TestNewConfigMissingParameter(t *testing.T) {
 			},
 			"*backends.Vault",
 		},
+		{
+			map[string]interface{}{
+				"AVP_TYPE":      "vault",
+				"AVP_AUTH_TYPE": "token",
+				"VAULT_TOKEN":   "token",
```

Comment:
This test is to make sure the backend does not instantiate, so you will want to leave off the VAULT_TOKEN

Reply:
Done!